LambdaSchool Site/app.js

```diff
@@ -0,0 +1,5 @@
+const button = document.getElementById('list-item dropdown');
```

Comment:
I'm glad you were able to get to the "Javascript" section of this sprint. This will prove much easier to accomplish with React etc.